grpc-google-cloud-datastore-v1/pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.api.grpc</groupId>
+  <artifactId>grpc-google-cloud-datastore-v1</artifactId>
+  <version>2.17.6-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-datastore-v1:current} -->
+  <name>grpc-google-cloud-datastore-v1</name>
+  <description>GRPC library for google-cloud-datastore</description>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-datastore-parent</artifactId>
+    <version>2.17.5</version><!-- {x-version-update:google-cloud-datastore:current} -->
```

Comment:
one last SNAPSHOT update to latest parent

Reply:
Fixed